docs/security/trust/content_trust.md

```diff
@@ -152,14 +152,36 @@ In an environment where `DOCKER_CONTENT_TRUST` is set, you can use the
 `--disable-content-trust` flag to run individual operations on tagged images
 without content trust on an as-needed basis.
 
-```bash
-$  docker pull --disable-content-trust docker/trusttest:untrusted
+Consider the following Dockerfile that uses an untrusted base image:
+
+```
+$  cat Dockerfile
+FROM docker/trusttest:latest
+RUN echo
+```
+
+In order to build a container successfully using this Dockerfile, one can do:
+
+```
+$  docker build --disable-content-trust -t <username>/nottrusttest:testing .
```

Comment:
@cyli I have made the changes we discussed above. Lemme know if you have any further comments.

PS: Haven't changed the image names, though. Lemme know if we wanna do that. If yes, we might even want to change/add the corresponding image to the docker registry as I mentioned in my above comment.